frontend/components/MultiColumnContent.vue

```diff
@@ -0,0 +1,33 @@
+<template>
```

Comment:
If we have different number of list elements in certain section, that breaks the flow of elements. (I'll provide screenshot in slack)

Reply:
fixed